src/basic/process-util.h

```diff
@@ -53,6 +54,8 @@ int get_process_umask(pid_t pid, mode_t *ret);
 
 int container_get_leader(const char *machine, pid_t *pid);
 
+int get_namespace_leader(pid_t pid, NamespaceType type, pid_t *ret);
```

Comment:
Maybe name this `namespace_get_leader()` since the above is named `container_get_leader()`?